File: Iserv.Niis.Model/Models/Payment/PaymentUseDto.cs
namespace Iserv.Niis.Model.Models.Payment
{
    public class PaymentUseDto
    {
        public int? PaymentId { get; set; }
        public int? PaymentInvoiceId { get; set; }
        public decimal Amount { get; set; }
        public int? CreateUserId { get; set; }
        public string AmountWithoutNds { set; get; }
        public string Description { get; set; }
    }
}

File: Iserv.Niis.Angular/src/app/payments-journal/models/payment-use.dto.ts
export class PaymentUseDto {
  public paymentId: number;
  public paymentInvoiceId: number;
  public amount: number;
  public createUserId: number;
  public description: string;

  constructor(init?: Partial<PaymentUseDto>) {
    Object.assign(this, init);
  }
}
